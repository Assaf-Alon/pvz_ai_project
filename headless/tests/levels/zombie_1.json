{
    "10": [["normal", 1]]
}
