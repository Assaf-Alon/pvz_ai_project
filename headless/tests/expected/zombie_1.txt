[100] Zombie being spawned at lane 1
[101] Sun autospawned. Total: 75.
[147] Zombie in (1, 9) moved to (1, 8).
[194] Zombie in (1, 8) moved to (1, 7).
[202] Sun autospawned. Total: 100.
[241] Zombie in (1, 7) moved to (1, 6).
[288] Zombie in (1, 6) moved to (1, 5).
[303] Sun autospawned. Total: 125.
[335] Zombie in (1, 5) moved to (1, 4).
[382] Zombie in (1, 4) moved to (1, 3).
[404] Sun autospawned. Total: 150.
[429] Zombie in (1, 3) moved to (1, 2).
[476] Zombie in (1, 2) moved to (1, 1).
[505] Sun autospawned. Total: 175.
[523] Zombie in (1, 1) moved to (1, 0).
[570] Zombie in (1, 0) attempted entering the house.
[570] Zombie in (1, 0) triggered a lawnmower.
[570] Lawnmower in (1, 0) attacked.
[570] Zombie in (1, 0) was damaged. HP: -9818.
[570] Zombie in (1, 0) was killed.
[570] You've won!
